packages/renderer/src/lib/dashboard/DashboardPage.svelte

```diff
@@ -1,18 +1,205 @@
 <script lang="ts">
-import { NavPage } from '@podman-desktop/ui-svelte';
+import { ListOrganizer, type ListOrganizerItem, NavPage, tablePersistence } from '@podman-desktop/ui-svelte';
+import { onMount } from 'svelte';
+import { SvelteMap } from 'svelte/reactivity';
 
-import { dashboardPageRegistry } from '/@/stores/dashboard/dashboard-page-registry.svelte';
+import {
+  convertFromListOrganizerItems,
+  dashboardPageRegistry,
+  type DashboardPageRegistryEntry,
+  defaultSectionNames,
+  setupDashboardPageRegistry,
+} from '/@/stores/dashboard/dashboard-page-registry.svelte';
 
 import NotificationsBox from './NotificationsBox.svelte';
+
+// Dashboard section configuration managed by dashboard page registry
+let dashboardSections = $state<ListOrganizerItem[]>([]);
+// eslint-disable-next-line svelte/no-unnecessary-state-wrap
+let dashboardOrdering = $state(new SvelteMap<string, number>());
```

Comment:
```suggestion
let dashboardOrdering = new SvelteMap<string, number>();
```

question: do we really need to wrap it in a state? I don't think so 🤔

Reply:
Well without it when you reshuffle it wont update :(